tools/MINGW-packages/mingw-w64-openimageio/PKGBUILD

```diff
@@ -33,22 +33,19 @@ source=(${_realname}-${pkgver}.tar.gz::https://github.com/OpenImageIO/oiio/archi
         oiio-2.1.10-boost.diff
         oiio-2.0.8-invalidatespec.patch
         oiio-2.2.14-libraw.diff
-        oiio-2.4.13.0-thread-shutdown.patch # Remove when updating to a future 2.5 release that has these changes.
```

Comment:
> It looks to be caused by the oiio-2.4.13.0-thread-shutdown.patch. You should be able to remove this patch since you are updating to 2.5.7, but you need to make sure that code in the openfx-io plugin still calls default_thread_pool_shutdown() when the plugin is unloaded.

done updating.